policyengine_us/variables/gov/states/la/tax/income/exemptions/la_disability_income_exemption.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class la_disability_income_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana disability income exemption"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.legis.la.gov/legis/Law.aspx?d=102133"  # (B)
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        amount = parameters(
+            period
+        ).gov.states.la.tax.income.exemptions.disability_income
+        disability_exemption = min_(
```

Comment:
does the tax form specify any disability benefits?

Reply:
past comment (aug 18) mentioned the disability part